civictechprojects/static/css/partials/_MyProjectCard.scss

```diff
@@ -16,14 +16,9 @@
 
 .MyProjectCard-projectName {
   font-weight: bold;
-}
-
-.MyProjectCard-table {
-  background-color: $color-background-light;
-  border: solid 1px $color-grey-frame-border;
-  margin: 20px auto;
-  padding: 16px;
-  width: 98%;
+  &.text-overflow-break {
```

Comment:
This works - but to save a few lines of CSS how about using the built-in bootstrap utility class `text-break` to do the same thing? ([ref](https://getbootstrap.com/docs/4.4/utilities/text/#word-break))

Reply:
Replaced with the built-in bootstrap utility class `text-break`